### PR TITLE
ZFIN-8286: Working config for Jenkins under docker.

### PR DESCRIPTION
**Pull request by zfinryanm**

### CODE REVIEW

**Review by rtaylorzfin — APPROVED** [review]
Looks good to me

File: server_apps/jenkins/jobs/Check-1-To-Many-ENSDARG-ZFIN/config.xml [hunk 1]
```diff
@@ -97,11 +97,5 @@
     </hudson.plugins.emailext.ExtendedEmailPublisher>
   </publishers>
   <buildWrappers>
-    <EnvInjectBuildWrapper plugin="envinject@1.89">
```
Comment (cmpich) [inline server_apps/jenkins/jobs/Check-1-To-Many-ENSDARG-ZFIN/config.xml:100]: I thought we need this plugin, don't we?

Comment (zfinryanm) [reply server_apps/jenkins/jobs/Check-1-To-Many-ENSDARG-ZFIN/config.xml:100]: We do but it wasn't working correctly under Jenkins 2.375. Not with docker, at least. I switched it to being injected globally and that worked, so we don't need it set on each individual job.

Comment (zfinryanm) [reply server_apps/jenkins/jobs/Check-1-To-Many-ENSDARG-ZFIN/config.xml:100]: See <globalNodeProperties> in the global config.xml.

**Review by cmpich — APPROVED** [review]
Looks good to me. Need to test it out once it is in main.

**Pull request merged**